ext/readline/tests/readline_error.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
+readline - test passing too many args returns false
+--SKIPIF--
+<?php if (!extension_loaded("readline")) die("skip"); ?>
+--FILE--
+<?php
+var_dump(readline("test", "test"));
+?>
+--EXPECTF--
+Warning: readline() expects at most 1 parameter, 2 given in %s on line %d
```

Comment:
This appears to be an unnecessary test of ZPP

Reply:
How would you recommend best to avoid this. Obviously I could suppress the error, but that seems gross, or tell it to not output errors? Unless there is another test declaration that I don't know about (probably!)

Reply:
@liam-wiltshire see also this discussion https://externals.io/message/103886#103989 for more info about this. Thank you.